scripts/postinstall.sh

```diff
@@ -1,4 +1,9 @@
-manage="${VENV}/bin/python ${INSTALLDIR}/${REPO}/manage.py"
+#!/bin/bash
 
-$manage syncdb --noinput --migrate
-$manage collectstatic --noinput
+# # Exit on errors from here.
+set -e
+
+composefile="${INSTALLDIR}/mama-ng-deploy/docker-compose.yml"
+
+docker-compose -f $composefile build
+docker-compose -f $composefile up -d
```

Comment:
I'm not sure if docker-compose would get confused by this or not, since supervisor will be running `docker-compose up`. I don't think that you need these lines, we just want supervisor to make sure that the repo exists on the server.